db/migrations/mysql/000012_create_posts.up.sql

```diff
@@ -0,0 +1,242 @@
+CREATE TABLE IF NOT EXISTS Posts (
+    Id varchar(26) NOT NULL,
+    CreateAt bigint(20) DEFAULT NULL,
+    UpdateAt bigint(20) DEFAULT NULL,
+    DeleteAt bigint(20) DEFAULT NULL,
+    UserId varchar(26) DEFAULT NULL,
+    ChannelId varchar(26) DEFAULT NULL,
+    RootId varchar(26) DEFAULT NULL,
+    ParentId varchar(26) DEFAULT NULL,
+    OriginalId varchar(26) DEFAULT NULL,
+    Message text,
+    Type varchar(26) DEFAULT NULL,
+    Props text,
+    Hashtags text,
+    Filenames text,
+    PRIMARY KEY (Id)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+SET @preparedStatement = (SELECT IF(
+    (
+        SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name = 'Posts'
+        AND table_schema = DATABASE()
+        AND column_name = 'FileIds'
+    ) > 0,
+    'SELECT 1',
+    'ALTER TABLE Posts ADD FileIds varchar(150);'
```

Comment:
a `CreateColumnIfNotExists` is used here with the default value of `[]`. Would we want to keep that default?

Reply:
Yeap, totally missed that.